services/agent/Dockerfile

```diff
@@ -118,6 +127,9 @@ ENV PYTHONOPTIMIZE=TRUE
 
 WORKDIR /home/scu
 
+# ensure home folder is read/writable for user scu
+RUN chown -R scu /home/scu
```

Comment:
Why is this necessary? Because the Dockerfile copies files in the scu home as root? You usually do `COPY --chown`
Also the entrypoing scripts usually mess around additionally

Reply:
the entrypoint.sh looks for all files with user=scu and changes the user/right according to the mounted folders right, but it does not check the /home/scu folder access itself.